plugins/plugin-docker/che-plugin-docker-machine/src/test/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProviderTest.java

```diff
@@ -70,14 +70,15 @@
 
 @Listeners(MockitoTestNGListener.class)
 public class DockerInstanceProviderTest {
-    private static final String PROJECT_FOLDER_PATH = "/projects";
-    private static final String CONTAINER_ID        = "containerId";
-    private static final String WORKSPACE_ID        = "wsId";
-    private static final String MACHINE_ID          = "machineId";
-    private static final String MACHINE_NAME        = "machineName";
-    private static final String USER_TOKEN          = "userToken";
-    private static final String USER_NAME           = "user";
-    private static final int    MEMORY_LIMIT_MB     = 64;
+    private static final String  PROJECT_FOLDER_PATH = "/projects";
+    private static final String  CONTAINER_ID        = "containerId";
+    private static final String  WORKSPACE_ID        = "wsId";
+    private static final String  MACHINE_ID          = "machineId";
+    private static final String  MACHINE_NAME        = "machineName";
+    private static final String  USER_TOKEN          = "userToken";
+    private static final String  USER_NAME           = "user";
+    private static final int     MEMORY_LIMIT_MB     = 64;
+    private static final boolean locallySnapshot     = true;
```

Comment:
new name should be snapshotUseRegistry to be more accurate with field rename in tested class